### PR TITLE
Skipping indentation on the first paragraph

### PR DESCRIPTION
**Pull request by madmurphy**
Added a sample usage of

``` css
p {
	text-align: justify;
	margin: 1em 0 0 0;
}

p + p {
	text-indent: 1em;
	margin: 0;
}
```

### CODE REVIEW

**Comment by CreaTorAlexander** [comment]
@madmurphy If there is an issue related to your PR, add a reference to the issue which your PR will close by #<issuenumber>

**Comment by madmurphy** [comment]
@CreaTorAlexander

Thank you for reaching out. There are no issues, I have only added an example for skipping indentation on the first paragraph. I hope that the example can be useful.

File: files/en-us/web/css/text-indent/index.html [hunk 1]
```diff
@@ -84,6 +84,61 @@ <h4 id="Result">Result</h4>
...
+<h4 id="CSS">CSS</h4>
+
+<pre class="brush: css">p {
+&nbsp;&nbsp;&nbsp;&nbsp;text-align: justify;
+&nbsp;&nbsp;&nbsp;&nbsp;margin: 1em 0 0 0;
+}</p>
+<p>p + p {
+&nbsp;&nbsp;&nbsp;&nbsp;text-indent: 1em;
+&nbsp;&nbsp;&nbsp;&nbsp;margin: 0;
+}</pre>
```
Comment (wbamberg) [inline files/en-us/web/css/text-indent/index.html:136]:
```suggestion
<pre class="brush: css">p {
    text-align: justify;
    margin: 1em 0 0 0;
}

p + p {
    text-indent: 1em;
    margin: 0;
}</pre>
```

File: files/en-us/web/css/text-indent/index.html [hunk 2]
```diff
@@ -84,6 +84,61 @@ <h4 id="Result">Result</h4>
 
 <p>{{ EmbedLiveSample('Simple_indent','100%','100%') }}</p>
 
+<h3 id="Skipping_indentation_on_the_first_paragraph">Skipping indentation on the first paragraph</h3>
+
+<p>A common typographic practice when paragraph indentation is present is to skip the indentation for the first paragraph. As the <em>The Chicago Manual of Style</em> puts it, &ldquo;the first line of text following a subhead may begin flush left or be indented by the usual paragraph indention.&rdquo;</p>
+
+<p>Treating differently first paragraphs from the other paragraphs can be done using the <a href="/en-US/docs/Web/CSS/Adjacent_sibling_combinator">adjacent sibling combinator</a>, as in the following example:</p>
```
Comment (wbamberg) [inline files/en-us/web/css/text-indent/index.html:91]:
```suggestion
<p>Treating first paragraphs differently from subsequent paragraphs can be done using the <a href="/en-US/docs/Web/CSS/Adjacent_sibling_combinator">adjacent sibling combinator</a>, as in the following example:</p>
```

File: files/en-us/web/css/text-indent/index.html [hunk 3]
```diff
@@ -84,6 +84,61 @@ <h4 id="Result">Result</h4>
...
+&nbsp;&nbsp;&nbsp;&nbsp;margin: 1em 0 0 0;
+}</p>
+<p>p + p {
+&nbsp;&nbsp;&nbsp;&nbsp;text-indent: 1em;
+&nbsp;&nbsp;&nbsp;&nbsp;margin: 0;
+}</pre>
+
+<h4 id="Result">Result</h4>
+
+<p>{{ EmbedLiveSample('Skipping_indentation_on_the_first_paragraph','100%','100%') }}</p>
```
Comment (wbamberg) [inline files/en-us/web/css/text-indent/index.html:140]:
```suggestion
<p>{{ EmbedLiveSample('Skipping_indentation_on_the_first_paragraph','','500px') }}</p>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
This is a nice example, thank you @madmurphy !

I had a few comments. Also I wonder if it's worthwhile making one of the example sections three paragraphs long, to emphasize that it's *all paragraphs after the first*, and not, say, just the paragraph immediately following the first?

**Comment by madmurphy** [comment]
@wbamberg That is great, thank you!

**Comment by madmurphy** [comment]
Any news on this PR?

**Comment by wbamberg** [comment]
@madmurphy , it looks like the changes I asked for weren't made, although the comments were marked as resolved. Was that your intention?

**Comment by madmurphy** [comment]
@wbamberg I must have misunderstood how the change request on GitHub works, I thought that by marking them as “resolved” they would be automatically merged. All done now.

**Review by wbamberg — APPROVED** [review]
Looks great @madmurphy , thanks for this!

**Pull request merged**